docs/how-to-guides/upgrade-bootstrap-kubelet.md

```diff
@@ -0,0 +1,113 @@
+# Upgrading bootstrap kubelet
+
+## Contents
+
+- [Introduction](#introduction)
+- [Steps](#steps)
+  - [Step 1: Drain the node](#step-1-drain-the-node)
+  - [Step 2: Find out the IP and SSH](#step-2-find-out-the-ip-and-ssh)
+  - [Step 3: Upgrade kubelet on node](#step-3-upgrade-kubelet-on-node)
+  - [Step 4: Verify](#step-4-verify)
+- [Caveats](#caveats)
+
+## Introduction
+
+[Kubelet](https://kubernetes.io/docs/reference/command-line-tools-reference/kubelet/) is a daemon
+that runs on every node and it is responsible for managing Pods on the node.
+
+Lokomotive cluster runs two different sets of kubelet processes. Initially, **bootstrap** kubelet
+configured on the node as a `rkt` pod joins the cluster, and then kubelet pod managed using
+DaemonSet (self-hosted kubelet) takes over the bootstrap kubelet. Self-hosted kubelet allows
+seamless updates between Kubernetes patch version for nodes and also to tune nodes configuration
+using tools like `kubectl`.
+
+At the time of writing `lokoctl` cannot update bootstrap kubelet, so this document explains how to
+perform this operation manually.
+
+## Steps
+
+Perform the following steps on each node, one node at a time.
```

Comment:
```suggestion
Perform the following steps on each node, one at a time.
```

Reply:
I believe `one node at a time` makes it more explicit.